src/modules/pokemons.js

```diff
@@ -0,0 +1,53 @@
+import unlike from '../images/unlike.png';
+import { displayPopup } from './popUp.js';
+import API from './api_data.js';
+
+const api = new API();
+const pokemonURL = api.urls.pokemons;
+
+let numberOfItems = 0;
+
+const cards = document.body.querySelector('.cards');
+cards.addEventListener('click', (e) => {
+  if (e.target && e.target.className === 'comment') {
+    const poke = Number(e.target.id - 1);
+    const url = `${pokemonURL}${poke}`;
+    displayPopup(url, poke);
+  }
+});
+
+const pokemons = async (url) => {
+  await fetch(url).then((response) => response.json())
+    .then((data) => data.results)
+    .then((obj) => Object.entries(obj))
+    .then((arr) => {
+      numberOfItems = arr.length;
+      arr.forEach((element) => {
+        const id = Number(element[0]) + 1;
+        let pokemonName = element[1].name;
+        pokemonName = pokemonName[0].toUpperCase() + pokemonName.slice(1);
+        fetch(element[1].url).then((response) => response.json())
+          .then((data) => data.sprites.other['official-artwork'].front_default)
+          .then((src) => {
+            const card = `<div class="card">
+            <img class="pokemon-img" id=${id} src="${src}" alt="Pokemons">
+            <div class="card-title">
+            <h2>${pokemonName}</h2>
+            <img id="likeId${id + 1}" class="likeBtn" src=${unlike} alt="like_icon">
+            </div> 
+            <p class="like-count" id="like-count${id + 1}">0 Likes</p>
+            <div class="buttons">
+              <button type="button" id="${id + 1}" class="comment">Comments</button>
```

Comment:
- [ ] I am able to see the comments only on the mobile view. I cannot see them on the desktop view. Even on the mobile view, I find it hard to read the comments. Please ensure that the comments written can be seen clearly as seen in the wireframe :+1: 

### View of comments on the mobile screen
![image](https://github.com/JuanDiegoGuerra/Module_2-Capstone/assets/45460192/fe550cc5-893a-466c-802a-f54f3b35ce6d)

### View of comments on the desktop view
![image](https://github.com/JuanDiegoGuerra/Module_2-Capstone/assets/45460192/ad2a8e24-b6ae-4e11-8cf0-4a334973a9ef)